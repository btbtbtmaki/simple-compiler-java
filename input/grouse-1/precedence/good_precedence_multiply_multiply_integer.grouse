main {

  print 10 * 2 * 19 nl;
  print 52 * 53 * 47 nl;
  print 93 * 31 * 9 nl;
  print 49 * 41 * 19 nl;
  print 24 * 8 * 35 nl;
  print 73 * 45 * 44 nl;
  print 88 * 36 * 46 nl;
  print 18 * 57 * 45 nl;
  print 72 * 71 * 73 nl;
  print 78 * 45 * 16 nl;
  print 11 * 66 * 53 nl;
  print 51 * 42 * 42 nl;

  print 24 * 57 * 58 nl;
  print 86 * 29 * 54 nl;
  print 55 * 11 * 19 nl;
  print 9 * 82 * 60 nl;
  print 7 * 65 * 65 nl;
  print 15 * 66 * 11 nl;
  print 19 * 46 * 45 nl;
  print 26 * 32 * 73 nl;
  print 41 * 69 * 83 nl;
  print 94 * 30 * 67 nl;
  print 25 * 1 * 21 nl;
  print 1 * 56 * 51 nl;

  print 9 * 80 * 30 * 82 nl;
  print 13 * 74 * 98 * 7 nl;
  print 49 * 72 * 54 * 44 nl;
  print 15 * 99 * 47 * 39 nl;
  print 14 * 79 * 61 * 87 nl;
  print 44 * 20 * 97 * 54 nl;
  print 29 * 79 * 90 * 4 nl;
  print 78 * 67 * 39 * 87 nl;
  print 56 * 89 * 24 * 95 nl;
  print 48 * 23 * 10 * 16 nl;
  print 16 * 49 * 73 * 7 nl;
  print 98 * 60 * 40 * 65 nl;

  print 88 * 95 * 94 * 91 nl;
  print 30 * 77 * 53 * 4 nl;
  print 57 * 90 * 92 * 22 nl;
  print 20 * 27 * 91 * 90 nl;
  print 43 * 81 * 70 * 52 nl;
  print 21 * 90 * 67 * 36 nl;
  print 26 * 71 * 81 * 55 nl;
  print 50 * 24 * 37 * 27 nl;
  print 21 * 3 * 57 * 89 nl;
  print 90 * 58 * 47 * 75 nl;
  print 74 * 77 * 58 * 61 nl;
  print 36 * 1 * 98 * 66 nl;

}

