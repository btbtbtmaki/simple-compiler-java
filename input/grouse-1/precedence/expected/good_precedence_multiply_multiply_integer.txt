380
129532
25947
38171
6720
144540
145728
46170
373176
56160
38478
89964
79344
134676
11495
44280
29575
10890
39330
60736
234807
188940
525
2856
1771200
659932
8382528
2722005
5869542
4609440
824760
17731818
11363520
176640
400624
15288000
71511440
489720
10383120
4422600
12678120
4558680
8223930
1198800
319599
18400500
20159524
232848
